Add citation page and remove duplicate info

micokoch opened the pull request:
This PR:
- Adds a separate citation page, since it is currently buried near the end of the long "about" section and is not easy or intuitive to find.
- Removed that section and the duplicate presentations section from the "about" page.

annakrystalli commented:
Unrelated question. Do we have a preview set up for PRs in this site? If not, maybe we should look into to setting one up?

micokoch commented on cite.qmd:
```diff
@@ -0,0 +1,14 @@
+
+### Citation
+
+The Consortium of Infectious Disease Modeling Hubs. The hubverse: open tools for collaborative modeling. GitHub release {{< var version >}}, {{< var version_date >}} URL: https://github.com/hubverse-org
```
Hi Anna, that makes sense. I went and checked, and the citation in hubDocs has a hard-coded write-up for the version and date. For practical purposes, the citation in the hubverse is also hard-coded. I'm not sure how the citation format came about, and it's certainly something we could discuss at the meeting.

micokoch commented:
Hi @annakrystalli - I have addressed the issues you brought up by opening separate issues:
1. An [issue on creating a preview in the PR](https://github.com/hubverse-org/hubverse-site/issues/65).
2. An [issue to update the citation page on hubDocs](https://github.com/hubverse-org/hubDocs/issues/411).
3. An [issue to create a script that imports the citation page from hubDocs to the hubverse site](https://github.com/hubverse-org/hubverse-site/issues/66).

Since these are all separate issues that will be addressed in separate PRs, I was hoping you could approve this PR, and the hubverse site citation page will serve as a placeholder until I've created the script (the last of the 3 PRs).

Thank you!

annakrystalli commented:
Thanks @micokoch Approved!
